bundles/audience_targeting.rst

```diff
@@ -0,0 +1,109 @@
+AudienceTargetingBundle
+=======================
+
+If you want to display different content on the same URL based on some
+characteristics of the user, you can use the AudienceTargetingBundle for that.
+
+It allows the content manager to define the audience target groups on his own.
```

Comment:
It allows the content manager to define audience target groups on his own. Each target group has one or multiple rules which are used to determine the target group of the visitor.